### DISCUSSION TITLE
Unknown fragment in vscode's graphql extension (LSP) when using client-preset with fragment masking

### DISCUSSION THREAD

**Discussion by eddiew** [dotansimha/graphql-code-generator#9748] [Q&A]
I'm using graphql-code-generator with client-preset, and overall it's working well for me. The types are generated correctly and I'm able to use fragment masking as recommended in the guide.

My issue is just related to tooling. I'm using VSCode's graphql extension to get auto-completion for graphql queries, and for some reason it can't find the fragment definitions, so most of my files have these "errors" in them.

<img width="774" alt="Screenshot 2023-11-05 at 3 44 15 AM" src="https://github.com/dotansimha/graphql-code-generator/assets/1779397/6f4b54d8-e3c0-48e7-8fdf-85a926c6e053">

I've looked thru existing discussions a bunch of times and couldn't find anything on this. Maybe someone can help me?

**Comment by Urigo** [comment]
Thank you for reporting, can you share a reproduction we can try to run with the extension?

**Comment by eddiew** [comment]
Sure, I'll work on finding a minimal repro. So I take it that not many
others have run into this issue then?

**Comment by eddiew** [comment]
Found the solution from this comment thread: https://github.com/dotansimha/graphql-code-generator/discussions/8554#discussioncomment-7490860

Basically, my problem was because I was using `include` instead of `documents` in my graphqlrc to specify which files to look at
